packages/shared/src/variants/quantum.ts

```diff
@@ -49,9 +49,16 @@ class BadukHelper {
 export class QuantumGo extends AbstractGame<NewBadukConfig, QuantumGoState> {
   subgames: BadukHelper[] = [];
   quantum_stones: Coordinate[] = [];
+  
+  public sgfContent: string = '';
+  private moves: string[] = [];
+  public moveNumber: number = 0;
+
 
   constructor(config: BadukConfig) {
     super(isLegacyBadukConfig(config) ? mapToNewConfig(config) : config);
+    this.bootstrap("player Black","player white", 9, 6.5); 
+    // TODO: make sure that player names, board size, and komi arugments are dynamic with user input
```

Comment:
I'm okay with hard-coding `PB` and `PW` for now, but can we implement size and komi?  Both are available on `BadukConfig`, so it should be trivial to pass into `bootstrap()`.

Note we support rectangular boards and so does SGF, so `SZ` might look something like `SZ[${width}:${height}]`.

Reply:
It seems as though BadukConfig does not have the board property. When obtaining the width using this: `config.board.width`. I get the error: " Property 'board' does not exist on type 'BadukConfig'. Property 'board' does not exist on type 'LegacyBadukConfig'."

Reply:
The structure of the config has changed during the development process, that is why BadukConfig is of type LegacyBadukConfig or NewBadukConfig. We support both at the moment, because the database contains games with the old config. But we can use typeguard functions to distinguish between the cases.

How about this:

```
  constructor(config?: BadukConfig) {
    super(isLegacyBadukConfig(config) ? mapToNewConfig(config) : config);

    if (isGridBadukConfig(this.config)) {
      if (this.config.board.width >= 52 || this.config.board.height >= 52) {
        throw new Error("Baduk does not support sizes greater than 52");
      }
      this.bootstrap("player Black","player white", this.config.board.width, this.config.board.height, this.config.komi);
      [...]
    }
  }
```